jujugui/static/gui/src/app/app.js

```diff
@@ -1782,9 +1782,11 @@ YUI.add('juju-gui', function(Y) {
     },
 
     _displayShortcutsModal: function() {
+      const guiVersion = GUI_VERSION; // eslint-disable-line no-undef
       ReactDOM.render(
         <window.juju.components.ModalShortcuts
           closeModal={this._clearShortcutsModal.bind(this)}
+          guiVersion={guiVersion.version}
```

Comment:
this can just be
`guiVersion={window.guiVersion.version}` and then you don't need 1785